firebase-firestore/src/main/java/com/google/firebase/firestore/Query.java

```diff
@@ -445,15 +520,44 @@ private void validateOrderByFieldMatchesInequality(
     }
   }
 
+  /**
+   * Given an operator, returns the set of operators that cannot be used with it.
+   *
+   * <p>Operators in a query must adhere to the following set of rules: 1. Only one array operator
```

Comment:
To make a list use

```html
<ol>
<li>Only one array operator ...
<li>One one disjunctive ...
</ol>
```

Reply:
The list wrapper element is required (`<ol>` indicates an ordered list (aka numbered); use `<ul>` for unordered (aka bulleted)).

Reply:
Done for reals.